install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -0,0 +1,142 @@
+Aggregator serves a critical subset of Kubecost APIs, but will eventually be the default model for Kubecost and serve all APIs. Currently, Aggregator supports all major monitoring and savings APIs, and also budgets and reporting.
+
+{% hint style="info" %}
+Existing documentation for Kubecost APIs will use endpoints for non-Aggregator environments unless otherwise specified, but will still be compatible after configuring Aggregator.
+{% endhint %}
+
+AAggregator is designed to accommodate queries of large-scale datasets by improving API load times and reducing UI errors. It is not designed to introduce new functionality; it is meant to improve functionality at scale.
+
+Aggregator is currently free for all Enterprise users to configure, and is always able to be rolled back.
+
+## Configuring Aggregator
+
+### Prerequisites
+
+* Aggregator can only be configured in a Federated ETL environment
+* Must be using v1.107.0 of Kubecost or newer
+* Your *values.yaml* file must have set `kubecostDeployment.queryServiceReplicas` to its default value `0`. 
+
+### Tutorial
+
+Select from one of the two templates below and save the content as *aggregator.yaml*. This will be your configuration template required to set up Aggregator.
+
+Basic configuration:
+
+```
+kubecostAggregator:
+  replicas: 1
+  enabled: true
+  cloudCost:
+    enabled: true
+federatedETL:
+  federatedCluster: true
+kubecostModel:
+  containerStatsEnabled: true
+  federatedStorageConfigSecret: federated-store
+kubecostProductConfigs:
+  clusterName: YOUR_CLUSTER_NAME
+  cloudIntegrationSecret: cloud-integration
+  productKey:
+    enabled: true
+    key: YOUR_KEY
+prometheus:
+  server:
+    global:
+      external_labels:
+        cluster_id: YOUR_CLUSTER_NAME
+# when using managed identity/irsa, set the service account accordingly:
+serviceAccount:
+  create: false
+  name: kubecost-irsa-sa
+```
+
+Advanced configuration (for larger deployments):
+
+```
+kubecostAggregator:
+  replicas: 1
+  enabled: true
+  cloudCost:
+    enabled: true
+  env:
+    # governs parallelism of derivation step
+    # more threads speeds derivation, but requires significantly more 
+    # log level
+    # default: info
+    LOG_LEVEL: debug
+  aggregatorDbStorage:
+    # governs storage size of aggregator DB storage
+    # !!NOTE!! disk performance is _critically important_ to aggregator performance
+    # ensure disk is specd high enough, and check for bottlenecks
+    # default: 128Gi
+    storageRequest: 128Gi
+  # tracing, starts embedded jaeger pod
+  # port forward to 16686 on the pod to access jaeger UI 
+  # provides instrumented traces used for troubleshooting performance issues
+  jaeger:

```

Comment:
@bstuder99 in the interest of removing unneeded debugging stuff, maybe we leave this jaeger config out? no point in giving people access to advanced debugging without us guiding them. Would want to remove lines 80-86